src/backend/joanie/payment/tests/utils.py

```diff
@@ -0,0 +1,15 @@
+"""Utils for payment test suites"""
+
+
+def get_billing_address(address):
+    """
+    Return a billing address dictionnary from an Address instance
+    """
+    return {
```

Comment:
why not put it in factories.py and use a factory boy DictFactory to generate it?